pkg/snapshot/snapshotter/snapshotter_test.go

```diff
@@ -572,6 +572,97 @@ var _ = Describe("Snapshotter", func() {
 			})
 		})
 	})
+
+	Describe("Scenarios to take full-snapshot during startup", func() {
```

Comment:
Can you please add a test for `GetFullSnapshotMaxTimeWindow()`?

Reply:
added. PTAL